t/ui/18-tests-details.t

```diff
@@ -455,7 +455,7 @@ subtest 'misc details: title, favicon, go back, go to source view, go to log vie
     # way lead to unstability (see poo#94060)
     $driver->get('/tests/99946/downloads_ajax');
     like $driver->find_element_by_id('asset-list')->get_text,
-      qr/openSUSE-13.1-DVD-i586-Build0091-Media.iso[\n|\s]+openSUSE-13.1-x86_64.hda \(does not exist\)/, 'asset list';
+      qr/openSUSE-13.1-DVD-i586-Build0091-Media.iso (0 Byte)[\n|\s]+openSUSE-13.1-x86_64.hda \(does not exist\)/, 'asset list';
```

Comment:
I suppose you need to escape the braces.

Reply:
Thanks, fixed!